packages/indicators/src/Builder/ArithmeticBuilder/config/aggregation.ts

```diff
@@ -79,12 +78,12 @@ const validateAggregationDictionary = (
   aggregation: Record<string, unknown>,
   config: { formula: 'string'; parameters: { code: 'string' }[] },
 ) => {
-  const { formula, parameters = [] } = config;
+  const { formula } = config;
 
   // Validate keys
   const parser = getExpressionParserInstance();
   parser.getVariables(formula).forEach(variable => {
-    if (!(variable in aggregation) && !isParameterCode(parameters, variable)) {
+    if (!(variable in aggregation)) {
```

Comment:
In the past, we didn't throw an error if a parameter did not have an aggregation specified for it. This was because we had no way of aggregating parameter results. Aggregation was coupled with data fetching in Aggregator. which in turn only works with data elements (eg indicators) defined in the DB, and not for inline indicators (parameters).

Now that aggregation can be done independently from fetching, we can aggregate parameter results, which may actually prove handy.